packages/css-data/src/parse-css.test.ts

```diff
@@ -84,181 +84,181 @@ describe("Parse CSS", () => {
       }
     `;
     expect(parseCss(css)).toMatchInlineSnapshot(`
-{
-  "test": [
-    {
-      "property": "backgroundImage",
-      "value": {
-        "type": "layers",
-        "value": [
-          {
-            "type": "unparsed",
-            "value": "linear-gradient(180deg,hsla(0,0.00%,0.00%,0.11),white)",
-          },
+      {
```

Comment:
what changed here?

Reply:
nothing, just indentation, I probably shouldn't be using indentation according to the code or jest should be respecting it? idk it works with indentation that is correct according to code, so idk why just doesn't just do it automatically

Reply:
weird, maybe merge conflict resolution went south, fixed